docs/index.rst

```diff
@@ -5,66 +5,85 @@ Tahoe-LAFS
 .. Please view a nicely formatted version of this documentation at
    http://tahoe-lafs.readthedocs.io/en/latest/
 
-Contents:
+Tahoe-LAFS is a Free and Open decentralized storage system.
+It distributes your data across multiple servers.
+Even if some of the servers fail or are taken over by an attacker,
+the entire file store continues to function correctly,
+preserving your privacy and security.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :caption: Introduction
 
    about
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Using Tahoe-LAFS
+
    INSTALL
    running
-   magic-wormhole-invites
    configuration
-   architecture
-
+   servers
    frontends/CLI
-   frontends/webapi
    frontends/FTP-and-SFTP
    frontends/download-status
-
+   magic-wormhole-invites
+   anonymity-configuration
    known_issues
-   contributing
-   CODE_OF_CONDUCT
-   release-checklist
 
-   servers
+.. toctree::
+   :maxdepth: 1
+   :caption: Technical Notes
+
+   architecture
    helper
    convergence-secret
    garbage-collection
-
-   backdoors
-   donations
-   accepting-donations
-   expenses
-   cautions
+   filesystem-notes
+   key-value-store
+   frontends/webapi
    write_coordination
+   cautions
    backupdb
-
-   developer-guide
-   ticket-triage
-
-   anonymity-configuration
-
    nodekeys
    performance
    logging
    stats
 
-   desert-island
-   debian
-   windows
-   OS-X
-   build/build-pyOpenSSL
+.. toctree::
+   :maxdepth: 1
+   :caption: Specifications
 
    specifications/index
    proposed/index
-   filesystem-notes
-   historical/configuration
-   key-value-store
 
-Indices and tables
-==================
+.. toctree::
+   :maxdepth: 1
+   :caption: Contributing to Tahoe-LAFS
+
+   contributing
+   CODE_OF_CONDUCT
+   developer-guide
+   ticket-triage
+   release-checklist
+   desert-island
```

Comment:
Wouldn't desert-island be a part of installation?

Reply:
That is actually geared towards Tahoe-LAFS hackers, according to the document itself:

> Here’s the story: you leave for the airport in an hour, you know you want to do some Tahoe hacking on the flight. What can you grab right now that will let you install the necessary dependencies later, when you are offline?

I don't think it should be part of installation.  In the current commit (d91638d) after merging master, it is under installation, but I'll move it back under developer notes if you agree about that choice.

Reply:
I think we can keep it near installation. We need to rework it along the lines of "Installing Tahoe-LAFS completely offline". But until we get to revamping that, we can keep it under developer notes.

Reply:
Keeping the 'build' with the 'install' documentation offers the user two incompatible ways of installation.
This choice takes from the documentation's clarity. Let's not make readers reason about things they probably do not know about yet (which method is the one for me?  when should I switch from one guide to the other?  do I have to undo all my steps if it turns out to not be working for some reason?  can i switch between both ways in the middle?).

I would put the 'build from source' with the developer documentation, even if building is a legit way of installation. Please see [my comment and commit below](https://github.com/tahoe-lafs/tahoe-lafs/pull/1030#discussion_r1716088809).